### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by whispyy**
Apply common style in managers.

### CODE REVIEW

**Comment by antleblanc** [comment]
:straight_ruler: PR to `develop` instead?
:bowtie: Please pay attention with commit message convention.

**Comment by whispyy** [comment]
Oh okay. I need to know about PR cause I followed where the last PR pointed about. I mean this PR (https://github.com/ovh-ux/ovh-module-emailpro/pull/6). Moreover I supposed that module hadn't develop branch. So please tell me if I need to change or not :)

**Comment by AxelPeter** [comment]
https://github.com/ovh-ux/ovh-module-emailpro/pull/6 has not the right target branch. You can change to `develop` :)

File: src/emailpro/emailpro.controller.js [hunk 1]
```diff
@@ -264,6 +264,7 @@ angular.module("Module.emailpro.controllers").controller("EmailProCtrl", [
                     case "todo":
                     case "DONE":
                     case "done":
+                    case "success":
```
Comment (rfougier) [inline src/emailpro/emailpro.controller.js]: Make sure adding lowercase in swicth ?
(update cases after)

File: src/emailpro/emailpro.html [hunk 2]
```diff
@@ -42,23 +22,21 @@ <h1 class="d-inline-block mb-0" data-ng-bind="exchange.displayName"></h1>
                 </div>
                 <div class="form-group" data-ng-show="edit.active"
                      data-ng-class="{'has-error': formEditDisplayName.txtDisplayName.$dirty && formEditDisplayName.$invalid}">
-                    <div class="input-group">
-                        <input type="text" class="form-control input-lg" name="txtDisplayName" required
+                    <div class="oui-input-group">
```
Comment (rfougier) [inline src/emailpro/emailpro.html]: `oui-input` doesn't work with Boostrap class `has-error`

File: src/emailpro/external-contact/emailpro-external-contact.controller.js [hunk 3]
```diff
@@ -138,10 +140,11 @@ angular.module("Module.emailpro.controllers").controller("EmailProExternalContac
...
     $scope.deleteAccount = function () {
         $scope.resetAction();
-        EmailProExternalContacts.removeContact($stateParams.organization, $stateParams.productId, $scope.model.externalEmailAddress).then((data) => {
-            Alerter.alertFromSWS($scope.tr("exchange_tab_EXTERNAL_CONTACTS_configuration_contact_delete_success"), data, $scope.alerts.dashboard);
-        }, (data) => {
-            Alerter.alertFromSWS($scope.tr("exchange_tab_EXTERNAL_CONTACTS_configuration_contact_delete_fail"), data, $scope.alerts.dashboard);
+        EmailProExternalContacts.removeContact($stateParams.organization, $stateParams.productId, $scope.model.externalEmailAddress).then(() => {
+            $scope.setMessage($scope.tr("exchange_tab_EXTERNAL_CONTACTS_configuration_contact_delete_success"), { status: "success" });
+        }, (err) => {
+            err.status = "error";
```
Comment (rfougier) [inline src/emailpro/external-contact/emailpro-external-contact.controller.js]: use `_.set` ?

File: src/emailpro/emailpro.controller.js (hunk 1 above)
Comment (AxelPeter) [reply src/emailpro/emailpro.controller.js]: Why not use `toLowerCase()` (or `toUpperCase()`) on `failure.type` instead of covering both cases ?

Comment (whispyy) [reply src/emailpro/emailpro.controller.js]: Yes that's what I've done !

File: src/emailpro/emailpro.controller.js [hunk 4]
```diff
@@ -241,29 +243,22 @@ angular.module("Module.emailpro.controllers").controller("EmailProCtrl", [
...
-                    case "CANCELLED":
                     case "cancelled":
-                    case "PAUSED":
                     case "paused":
-                    case "ERROR":
                     case "error":
                         alertType = "alert alert-danger";
                         break;
-                    case "WAITING_ACK":
                     case "waitingAck":
```
Comment (AxelPeter) [inline src/emailpro/emailpro.controller.js]: `waitingack` if lowercased, no ?

Comment (whispyy) [reply src/emailpro/emailpro.controller.js]: yes

File: src/emailpro/emailpro.controller.js [hunk 5]
```diff
@@ -241,29 +243,23 @@ angular.module("Module.emailpro.controllers").controller("EmailProCtrl", [
...
-                    case "PAUSED":
                     case "paused":
-                    case "ERROR":
                     case "error":
                         alertType = "alert alert-danger";
                         break;
-                    case "WAITING_ACK":
-                    case "waitingAck":
-                    case "WARNING":
+                    case "waitingack"
```
Comment (rfougier) [inline src/emailpro/emailpro.controller.js]: missing `:`

File: src/emailpro/account/emailpro-account.controller.js [hunk 6]
```diff
@@ -80,8 +80,8 @@ angular.module("Module.emailpro.controllers").controller("EmailProTabAccountsCtr
                 $scope.loading = false;
                 $scope.accounts = accounts;
 
-                var act;
-                for (var i = 0, c = accounts.list.results.length; i < c; i++) {
+                let act;
+                for (let i = 0, c = accounts.list.results.length; i < c; i++) {
```
Comment (jleveugle) [inline src/emailpro/account/emailpro-account.controller.js]: We should use a `map` function

File: src/resources/i18n/emailpro/Messages_fr_FR.xml [hunk 7]
```diff
@@ -48,13 +48,13 @@
    <translation id="emailpro_tab_account_remove_title" qtlid="375465">Voulez-vous réellement supprimer le compte suivant de votre service E-mail Pro ?</translation>
    <translation id="emailpro_tab_account_remove_success" qtlid="375478">Le compte va être supprimé de votre service E-mail Pro.</translation>
 
-   <translation id="emailpro_tab_domain_modify_domain_tooltip">
-       L'antispam Vade Secure est compatible avec le mode non autoritatif (uniquement si vos comptes pop/imap sont hébergés par OVH). Dans ce cas, il est conseillé d'utiliser : <strong>{0}</strong>, <strong>{1}</strong>, <strong>{2}</strong> et <strong>{3}</strong>
-       <br /><br /> Si vous utilisez le mode non autoritatif avec des comptes e-mail non hébergés par OVH, vous devez utiliser : <strong>{4}</strong> (voir section domaines associés)
-   </translation>
-    <translation id="emailpro_tab_domain_manage">: gérer</translation>
+<translation id="emailpro_tab_domain_modify_domain_tooltip" qtlid="416397;416410;416410;169498;416423;416449">
```
Comment (jleveugle) [inline src/resources/i18n/emailpro/Messages_fr_FR.xml]: I don't understand theses change :(

`strong` + `qtlid` in a translation ? :o

**Pull request merged**